var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -183,3 +183,8 @@ def cmake_args(self):
                 args.append("-DENABLE_STATIC_LIBS=NO")
 
         return args
+
+    def test_ptls(self):
+      parseThat = which(join_path(self.prefix.bin, "parseThat"))
+      os.environ["DYNINSTAPI_RT_LIB"] = join_path(self.prefix, "lib", "libdyninstAPI_RT.so")
```

Comment:
Optional. Same here for joining `self.prefix` and `lib`, though we don't want an `os.sep` to replace the `.` in the library name.

```suggestion
      os.environ["DYNINSTAPI_RT_LIB"] = join_path(self.prefix.lib, "libdyninstAPI_RT.so")
```